futures-channel/tests/mpsc.rs

```diff
@@ -116,8 +116,6 @@ fn recv_close_gets_none() {
             Poll::Ready(Err(e)) => assert!(e.is_disconnected()),
         };
 
-        drop(&tx);
```

Comment:
Dropping a reference seems suspicious-- was this ever supposed to do something interesting?

Reply:
It was changed from a by-value drop to a by-ref drop in https://github.com/rust-lang/futures-rs/commit/fff7e7b09a60c73e303d6e757d20aeb0a6951d3d#diff-4198821522533ce7112d24aa37e3926dL132-R132, but even before that it seems to have been a useless drop just before the closure would have dropped it anyway.